scripts/globals/missions.lua

```diff
@@ -53,6 +53,17 @@ tpz.mission.logEx =
 
 tpz.mission.id =
 {
+    -----------------------------------
+    --  All Nations
+    -----------------------------------
+    ['nation'] =
+    {
+        MAGICITE    = 13,
+        VAHZL       = 14,
```

Comment:
VAHZL? How about ARCHLICH so everyone knows what it is at a glance.

Reply:
Yep you're totally right idk what I was thinking there, fixing now.